target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -275,6 +275,15 @@ yuncore,a770)
 	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth1"
 	ucidef_set_led_switch "lan" "LAN" "$boardname:green:lan" "switch0" "0x10"
 	;;
+yuncore,cpe830)
+	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth1"
+	ucidef_set_led_switch "lan" "LAN" "$boardname:green:lan" "switch0"
```

Comment:
That's not how it works. ucidef_set_led_switch needs a 5th argument which is a port mask. You could connect a network cable to the lan port on this router and execute the following command on your router:
```
swconfig dev switch0 show
```
and find out which of port1-port4 is link up.
If it's port 1, the port mask here should be 0x02, port2->0x04 port3-> 0x08 port4->0x10 respectively.

Reply:
# swconfig dev switch0 show
Global attributes:
	enable_vlan: 0
	ar8xxx_mib_poll_interval: 500
	ar8xxx_mib_type: 0
	enable_mirror_rx: 0
	enable_mirror_tx: 0
	mirror_monitor_port: 0
	mirror_source_port: 0
	arl_table: address resolution table
Port 0: MAC 44:d1:fa:75:20:65
Port 4: MAC 00:0e:c6:a4:84:38

Port 0:
	mib: MIB counters
RxGoodByte  : 290836 (284.0 KiB)
TxByte      : 62872 (61.3 KiB)

	pvid: 0
	link: port:0 link:up speed:1000baseT full-duplex txflow rxflow 
Port 1:
	mib: No MIB data
	pvid: 0
	link: port:1 link:down
Port 2:
	mib: No MIB data
	pvid: 0
	link: port:2 link:down
Port 3:
	mib: No MIB data
	pvid: 0
	link: port:3 link:down
Port 4:
	mib: MIB counters
RxGoodByte  : 63148 (61.6 KiB)
TxByte      : 292840 (285.9 KiB)

	pvid: 0
	link: port:4 link:up speed:100baseT full-duplex auto